src/test/java/de/tum/in/www1/artemis/FileUploadSubmissionIntegrationTest.java

```diff
@@ -356,6 +360,23 @@ public void getDataForFileUpload() throws Exception {
         assertThat(submission.getLatestResult().getFeedbacks().size()).as("No feedback should be returned for editor").isEqualTo(0);
     }
 
+    @Test
+    @WithMockUser(value = "student1")
```

Comment:
is there any reason why 'roles="USER''' is not added for newly created tests? - I was just curious about...

Reply:
If I understand this correctly, the USER role is the default in this case, so no need to clarify the role then. If someone else knows better feel free to correct me ^^'